### ISSUE TITLE
frida-7.1.7: Not installable (on OSX, ...)

### ISSUE THREAD

**Issue by jenisys** [frida/frida#143]
USING: Python 2.7.11 (with pip-8.1.1, setuptools-20.10.1)
Trying to install frida-7.1.7 (zip variant), leads to installation error:

```sh
$ sudo pip instal -U frida
Collecting frida
  Downloading frida-7.1.7.zip
  ...
Installing collected packages: frida
Running setup.py install for frida ... error
    Complete output from command 
  ...
    running install  
running build_ext
    querying pypi for available prebuilds
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/tmp/pip-build-XXXXX/frida/setup.py", line 161, in <module>
        'build_ext': FridaPrebuiltExt
    ...
    File "/private/tmp/pip-build-XXXXX/frida/setup.py", line 93, in build_extension
        egg_path = os.path.join(os.getcwd(), egg_filename)
    UnboundLocalError: local variable 'egg_filename' referenced before assignment
```

**Comment by oleavr** [comment]
Could you try again now? Should be fixed in frida 7.1.8.

**Comment by jenisys** [comment]
Looks good. Everything is fine again :smile:
